tests/test-hello-1.out

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
So I don't think we should be testing code - we're supposed to be testing outputs of printed things, not of code. Like, say later we decide to compile to another language or we change the formatting of the code we're outputting - we wouldn't want all the tests to break as a result